lib/trento/domain/cluster/cluster.ex

```diff
@@ -262,6 +267,26 @@ defmodule Trento.Domain.Cluster do
     }
   end
 
+  def execute(
```

Comment:
We should consider the scenario when a `DeregisterClusterHost` is triggered for a cluster that doesn't exist, doing nothing and returning an error in this case.
You might tell me that this won't happen as by now the Process manager is the unique one that runs this commands, but I can go myself and run the command in the console, or in the future the command might be executed by some automatic de-registration, right?

Reply:
This is already handled by line 221, every command except the registration ones are rejected if the cluster does not exist

Reply:
Totally right! I missed that piece of code